core/drivers/fabric-driver/server/server.ts

```diff
@@ -253,6 +253,22 @@ const configSetup = async () => {
     logger.info(`Load Event Subscriptions Status: ${status}`);
 };
 
+const monitorService = async () => {
+    const delayTime: number = parseInt(process.env.MONITOR_SYNC_PERIOD ? process.env.MONITOR_SYNC_PERIOD : '30');
+    const networkName = process.env.NETWORK_NAME ? process.env.NETWORK_NAME : 'network1';
+    const flagEnable = process.env.ENABLE_MONITOR === 'false' ? false : true;
+    if (flagEnable) {
+        logger.info("Starting monitor...");
+        logger.info("Monitor sync period: ", delayTime);
+    } else {
+        logger.info("Monitor disabled.");
+    }
+    while (flagEnable) {
```

Comment:
Is there a way to disable this dynamically, i.e., via a service request?

Reply:
Can be done by adding a new endpoint